### DISCUSSION TITLE
Hiding Open sandbox button

### DISCUSSION THREAD

**Discussion by crossLineApex** [codesandbox/codesandbox-client#5802] [Q&A] [answered]
When I get a link for my project in codesandbox it comes with a open sandbox which on click opens the ressulting code.
Is there a way to avoid that button ?

**Comment by smhutch** [comment, marked as answer]
@crossLineApex it sounds like you are using an embed link (Which you get by clicking "Share" then "Embed"). With these links there is no way to disable the "Open Sandbox" button.

If you would like to share a link to your Sandbox without this link, you can copy a link by clicking on **Share > Copy Sandbox link**

**Reply by 1kevinson** [reply]
Hello, please any update to the possibility to hide those buttons? 

![CleanShot 2024-12-07 at 13 29 35@2x](https://github.com/user-attachments/assets/8d13127d-1422-4191-be2e-d544e56c07fa)
